pic32/variants/Max32/Board_Defs.h

```diff
@@ -300,6 +300,10 @@ extern const uint32_t   digital_pin_to_cn_PGM[];
 #define _SER0_IPL_ISR	IPL2SOFT
 #define	_SER0_IPL		2
 #define	_SER0_SPL		0
+#define _SER0_TX_PIN    1
+#define _SER0_RX_PIN    0
+#define _SER0_RTS_PIN   18
```

Comment:
For the RTS and CTS pins, I don't think that any of our available UART driver code can use these definitions. I don't think we support hardware handshaking, do we?

In any case, I don't think it can hurt to define these, except if there were to be a library or sketch that checked for these pins to be defined, and if so, assumed that hardware handshaking was being used.